docker-storage-setup.sh

```diff
@@ -978,7 +976,7 @@ setup_extra_volume() {
 }
 
 setup_extra_lv_fs() {
-  [ -z "$CONTAINER_ROOT_LV_MOUNT_PATH" ] && [ -z "$CONTAINER_ROOT_LV_NAME" ] && return 0
+  [ -z "$RESOLVED_MOUNT_DIR_PATH" ] && [ -z "$CONTAINER_ROOT_LV_NAME" ] && return 0
```

Comment:
What are we gaining by this change? Checking for CONTAINER_ROOT_LV_MOUNT_PATH is simpler in my opinion as it is well documented. RESOLVVED_MOUNT_DIR_PATH is just an internal variable.